apps/dashboard/app/(app)/apis/[apiId]/settings/default-bytes.tsx

```diff
@@ -84,7 +84,7 @@ export const DefaultBytes: React.FC<Props> = ({ keyAuth }) => {
               render={({ field }) => (
                 <Input
                   className="max-w-sm"
-                  {...field}
```

Comment:
Removing this, means that the current saved bytes in the database is not displayed on load / refresh.

Reply:
@perkinsjr Pushed!

Reply:
the only problem is this 

https://www.loom.com/share/63dc66f52d8b4725a8c6c6fd91f238d8?sid=ee72d670-fa76-4a43-95f3-7ba63d19f453

I won't say this is a problem/issue per say. But backspacing fully won't get rid of the 0

Reply:
You need to fix that at the same time. The field should be empty if the user wants to remove the entire field and type a new number in. It certainly should be be `0` when nothing is in the field.

Reply:
It's 0 when there is nothing in the field. That's not the problem. If you try to type something after making it 0. It goes like `016 ` rather than `16. ` I am probably not articulating that well enough.

Reply:
Both are a problem 

It shouldn't be `0` when there is nothing in the field because technically the byte size default is never 0 when you don't have a default, nor should it be`016` when typing

Reply:
`016` is the only problem here. If it is anything below 8 save button will be disabled.